src/js/angular/core/services/jwt-auth.service.js

```diff
@@ -309,6 +307,13 @@ angular.module('graphdb.framework.core.services.jwtauth', [
                 });
             };
 
+            this.getSelectedRepo = function () {
```

Comment:
Why is this here? It duplicates a function in repositories service!

Reply:
replaced direct LocalStorageAdapter/LSKeys access with RepositoryStorage in jwt-auth.service and repositories.service.